src/test/java/com/integrivideo/Data.java

```diff
@@ -13,6 +13,7 @@
     public final static String CARDHOLDER_NAME = "Integri User";
     public final static String CARD_NUMBER = "378282246310005";
     public final static String CREATE_PROJECT_URL = BASE_URL + "app/projects/new";
+    public final static String DOWNLOAD_FOLDER = System.getProperty("user.dir").concat("\\target\\DownloadsIntegriVideo\\");
```

Comment:
Not sure that it is needed here as a variable (probably hardcode is enough). Also not sure whether file separators used will work for Unix and Windows